querybook/webapp/components/DataDoc/DataDoc.tsx

```diff
@@ -314,7 +315,8 @@ class DataDocComponent extends React.PureComponent<IProps, IState> {
                     index,
                     cellType,
                     context,
-                    meta
+                    meta,
+                    previousEngine
```

Comment:
you can bypass all of these prop passing with this:
```
const previousQueryCell = dataDoc.dataDocCells.slice(0, index).reverse().find(cell => cell.cell_type === 'query');
const previousCellEngineId = previousQueryCell?.meta?.engineId;
```

Reply:
Oh nice! You're right, I'll make the change